build.gradle

```diff
@@ -22,6 +22,7 @@ allprojects {
 
 checkstyle {
     maxWarnings = 0
+    ignoreFailures = false
```

Comment:
@Itay-Kirsh default is normally false

![Screenshot 2022-07-21 at 14 57 00](https://user-images.githubusercontent.com/78368309/180231803-0a1e9f1e-dc22-4a52-8899-c00bbbcf6a19.png)

Reply:
Also, we use severity as warning and not error in check style.xml. Would this setting really make any difference?